ui/src/main/webapp/components/MenuInput.stories.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useState, useEffect } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import MenuInput from './MenuInput';
+import { setUnifiedConfig } from '../util/util';
+import { UnifiedConfig } from '../types/config';
+
+interface MenuInputProps {
+    handleRequestOpen: (args: { serviceName: string; input?: string; groupName?: string }) => void;
+    config: UnifiedConfig;
+}
+
+function MenuInputWrapper(props: MenuInputProps) {
+    const [display, setDisplay] = useState(false);
+
+    useEffect(() => {
+        setUnifiedConfig(props.config);
+        setDisplay(true);
+    }, [props.config]);
+
+    return display ? (
+        <MenuInput handleRequestOpen={props.handleRequestOpen} />
+    ) : (
+        <span>loading</span>
+    );
+}
+
+const meta = {
+    component: MenuInputWrapper,
```

Comment:
Can't you use [Decorators](https://storybook.js.org/docs/react/writing-stories/decorators) instead?